Corona-Warn-App/src/main/java/de/rki/coronawarnapp/statistics/ui/homecards/StatisticsCardPaddingDecorator.kt

```diff
@@ -0,0 +1,38 @@
+package de.rki.coronawarnapp.statistics.ui.homecards
+
+import android.graphics.Rect
+import android.view.View
+import androidx.annotation.DimenRes
+import androidx.recyclerview.widget.RecyclerView
+import androidx.recyclerview.widget.RecyclerView.ItemDecoration
+
+class StatisticsCardPaddingDecorator(
+    @DimenRes val startPadding: Int,
+    @DimenRes val endPadding: Int = startPadding,
+    @DimenRes val cardDistance: Int = startPadding
+) : ItemDecoration() {
+
+    override fun getItemOffsets(outRect: Rect, view: View, parent: RecyclerView, state: RecyclerView.State) {
+        super.getItemOffsets(outRect, view, parent, state)
+
+        val itemPosition = parent.getChildAdapterPosition(view)
+
+        if (itemPosition == RecyclerView.NO_POSITION) return
+
+        val resources = parent.context.resources
+
+        if (itemPosition == 0) {
+            outRect.left = resources.getDimensionPixelSize(startPadding)
+        } else {
+            parent.adapter?.let {
+                if (itemPosition == it.itemCount - 1) {
+                    outRect.right = resources.getDimensionPixelSize(endPadding)
+                } else {
+                    val distance = resources.getDimensionPixelSize(cardDistance)
+                    outRect.left = distance / 2
+                    outRect.right = distance / 2
+                }
+            }
+        }
+    }
```

Comment:
maybe:
```
when(itemPosition){
}
``` 
is more readable

Reply:
Not sure due to the `adapter.?` but will try. :ok_hand:

Reply:
Done.